src/bitcoin-chainstate.cpp

```diff
@@ -70,7 +70,10 @@ int main(int argc, char* argv[])
 
 
     // SETUP: Chainstate
-    ChainstateManager chainman{chainparams};
+    const ChainstateManager::Options chainman_opts{
```

Comment:
nit: may be more pragmatic to keep name as 'params' or 'settings' since option made me think std::option.

Reply:
I don't mind either way! I'm following how `BlockAssembler::Options` and `CConnman::Options` do it, but if others think we should use `Params` or something else for these structs in the future, let me know in this thread, I can change it.

Reply:
fair point -- i personally think config/params/settings are better than options, but consistency is probably a overriding interest.

Reply:
Going to mark as resolved, feel free to reopen for futher convo!